### PR TITLE
clear achievement tracker

### PR DESCRIPTION
**Pull request by kevingranade**
SUMMARY: None

Windows travis build still breaking due to ordering issues.  Adding some fixups.

### CODE REVIEW

File: tests/memorial_test.cpp [hunk 1]
```diff
@@ -9,6 +9,7 @@
 #include <string>
 #include <vector>
 
+#inclyde "achievements.h"
```
Comment (ZhilkinSerg) [inline tests/memorial_test.cpp:12]:
```suggestion
#include "achievements.h"
```

File: tests/memorial_test.cpp [hunk 2]
```diff
@@ -9,6 +9,7 @@
 #include <string>
 #include <vector>
 
+#include "achievements.h"
```
Comment (ZhilkinSerg) [inline tests/memorial_test.cpp:12]:
```suggestion
#include "achievement.h"
```

**Comment by ZhilkinSerg** [comment]
Not crashing now, but here's your favorite:

```
-------------------------------------------------------------------------------

check_npc_behavior_tree

  Freezing

-------------------------------------------------------------------------------

behavior_test.cpp:154

...............................................................................

behavior_test.cpp:160: FAILED:

  CHECK( npc_needs.tick( &oracle ) == "wear_warmer_clothes" )

with expansion:

  "idle" == "wear_warmer_clothes"

behavior_test.cpp:166: FAILED:

  CHECK( npc_needs.tick( &oracle ) == "start_fire" )

with expansion:

  "idle" == "start_fire"
```

**Comment by kevingranade** [comment]
Well crap, yea more test ordering fragility most likely.

**Comment by ZhilkinSerg** [comment]
At last!

image

**Comment by ZhilkinSerg** [comment]
There are some OSX tests issues left, but I think it is good to go as it is:


```
-------------------------------------------------------------------------------
send_event_through_bus
-------------------------------------------------------------------------------
event_test.cpp:33
...............................................................................
event_test.cpp:41: FAILED:
  REQUIRE( sub.events.size() == 1 )
with expansion:
  0 == 1
-------------------------------------------------------------------------------
destroy_bus_before_subscriber
-------------------------------------------------------------------------------
event_test.cpp:49
...............................................................................
event_test.cpp:57: FAILED:
  CHECK( sub.events.size() == 1 )
with expansion:
  0 == 1
```

and

```
iteminfo_test.cpp:2350
...............................................................................
iteminfo_test.cpp:2355: FAILED:
  CHECK( item_info_str( iodine, crafting ) == "--\n" "You could use it to craft: " "<color_c_dark_gray>water purification tablet</color>" " and <color_c_dark_gray>antiseptic powder</color>\n" )
with expansion:
  "--
  You could use it to craft: <color_c_dark_gray>antiseptic powder</color> and
  <color_c_dark_gray>water purification tablet</color>
  "
  ==
  "--
  You could use it to craft: <color_c_dark_gray>water purification tablet</
  color> and <color_c_dark_gray>antiseptic powder</color>
  "
```

**Pull request merged**